chrome/content/zotero/xpcom/style.js

```diff
@@ -81,9 +81,41 @@ Zotero.Styles = new function() {
 		
 		// hidden dir
 		dir.append("hidden");
-		if(dir.exists()) i += _readStylesFromDirectory(dir, true);
+		if (dir.exists()) i += _readStylesFromDirectory(dir, true);
 		
 		Zotero.debug("Cached "+i+" styles in "+((new Date()).getTime() - start)+" ms");
+		
+		// transfer and reset "export.bibliographyLocale" pref value
+		var bibliographyLocale = '';
+		bibliographyLocale = Zotero.Prefs.get("export.bibliographyLocale");
+		if (bibliographyLocale) {
+			Zotero.Prefs.set("export.lastLocale", bibliographyLocale);
+			Zotero.Prefs.set("export.quickCopy.locale", bibliographyLocale);
```

Comment:
FYI, this might not always make sense. If the currently-selected quick-copy style has a default locale, then this may mismatch. Thus, we should not depend on this value directly being correct. (it can also be an empty string if bibliographyLocale was not set to begin with). That reminds me that we need to add all new preferences to defaults/preferences/zotero.js (I'll make a comment there as well as a reminder)